toolbox/log-purge/log-purge.cron

```diff
@@ -0,0 +1,11 @@
+# Cron job to clean up old files and empty directories from /usr/airflow/atd-airflow/logs
+# This will run nightly at 2 AM. The first command removes all files which have a modification
+# time greater than 30 days, and the second command removes all empty directories. Removed
+# files and directories are logged into /var/log which will get rotated by the system.
+
+0 2 * * * find /usr/airflow/atd-airflow/logs -type f -mtime +30 -exec rm -v {} \; \
```

Comment:
2 AM sound like a good time to do the rotation?

Reply:
sounds great!